From eef366956e2e4a689ab33a0d1f265eb15f749d8d Mon Sep 17 00:00:00 2001
From: Alejandro Perez <alex.perez-mendez@jisc.ac.uk>
Date: Mon, 6 Apr 2020 15:52:38 +0200
Subject: [PATCH] Add missing backslash that precluded server from starting

Server did not start, because it found an unterminated string. Adding the previous-existing backslash fixed it.
```
including configuration file /etc/raddb/mods-config/sql/main/postgresql/queries.conf
/etc/raddb/mods-config/sql/main/postgresql/queries.conf[505]: Parse error: Unterminated string
Errors reading or parsing /etc/raddb/radiusd.conf
```
---
 raddb/mods-config/sql/main/postgresql/queries.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/raddb/mods-config/sql/main/postgresql/queries.conf b/raddb/mods-config/sql/main/postgresql/queries.conf
index da82467aea5..b868238306a 100644
--- a/raddb/mods-config/sql/main/postgresql/queries.conf
+++ b/raddb/mods-config/sql/main/postgresql/queries.conf
@@ -502,7 +502,7 @@ accounting {
 					FramedInterfaceId = NULLIF('%{Framed-Interface-Id}', ''), \
 					DelegatedIPv6Prefix = NULLIF('%{Delegated-IPv6-Prefix}', '')::inet, \
 					AcctUpdateTime = ${....event_timestamp}, \
-					AcctSessionTime = COALESCE(%{%{Acct-Session-Time}:-NULL},
+					AcctSessionTime = COALESCE(%{%{Acct-Session-Time}:-NULL}, \
 						(${....event_timestamp_epoch} - EXTRACT(EPOCH FROM(AcctStartTime)))), \
 					AcctInputOctets = (('%{%{Acct-Input-Gigawords}:-0}'::bigint << 32) + \
 						'%{%{Acct-Input-Octets}:-0}'::bigint), \
